Content.Server/Tiles/FloorTileSystem.cs

```diff
@@ -49,8 +50,15 @@ private void OnAfterInteract(EntityUid uid, FloorTileComponent component, AfterI
                 if (physics.TryGetComponent(ent, out var phys) &&
                     phys.BodyType == BodyType.Static &&
                     phys.Hard &&
-                    (phys.CollisionLayer & (int) CollisionGroup.Impassable) != 0)
-                    return;
+                    (phys.CollisionLayer & (int) CollisionGroup.Impassable) != 0) 
+                    {
+                        // e.g. you can place tile under directional windows
+                        if (!TryComp<FixturesComponent>(ent, out var fixtures) || 
+                            fixtures.GetAABB(new Transform(new Vector2(0, 0), 0)).Contains(Box2.UnitCentered)) 
```

Comment:
This shouldnt be using Contains()
![image](https://user-images.githubusercontent.com/60196617/217417665-b04a3149-925f-4a26-b2fe-baf4d2284b54.png)

Reply:
Well idea was is that a floor tile is 1 by 1 square, so if entity fully contains the floor tile then you can't access plating.

Reply:
floor tiles are not guaranteed to be 1x1, no.